download_police_data.R

```diff
@@ -0,0 +1,14 @@
+# script for downloading slovenian police data  
```

Comment:
Start sentence with a capital letter. End it with a full stop. Add Slovenian translation.